hardeneks/cluster_wide/scalability/control_plane.py

```diff
@@ -22,3 +23,24 @@ def check_EKS_version(resources: Resources):
         return False
 
     return True
+
+#
+# check_kubectl_compression
+# checks all clusters in config for disable-compression flag set to true
+# if any cluster does not have setting, it returns False
+def check_kubectl_compression(resources: Resources):
+    kubeconfig = helpers.get_kube_config()
+    isSetCorrectly = True
+    for cluster in kubeconfig.get("clusters", []):
+        clusterName = cluster.get("name", "NoName")
```

Comment:
Can we check this for the current cluster only? Resources object should have the cluster name:

Something like:
```python
if clusterName == resources.cluster:
    if cluster.get("cluster", {}).get.......:    
```